osu.Game.Tournament/Screens/Gameplay/Components/MatchScoreDisplay.cs

```diff
@@ -127,21 +128,28 @@ protected override void UpdateAfterChildren()
 
         private class MatchScoreCounter : ScoreCounter
         {
+            private OsuSpriteText displayedSpriteText;
+
             public MatchScoreCounter()
             {
                 Margin = new MarginPadding { Top = bar_height, Horizontal = 10 };
-
-                Winning = false;
-
-                DisplayedCountSpriteText.Spacing = new Vector2(-6);
             }
 
             public bool Winning
             {
-                set => DisplayedCountSpriteText.Font = value
+                set => displayedSpriteText.Font = value
                     ? OsuFont.Torus.With(weight: FontWeight.Bold, size: 50, fixedWidth: true)
                     : OsuFont.Torus.With(weight: FontWeight.Regular, size: 40, fixedWidth: true);
             }
+
+            protected override OsuSpriteText CreateSpriteText()
+            {
+                displayedSpriteText = base.CreateSpriteText();
+                displayedSpriteText.Spacing = new Vector2(-6);
+                Winning = false;
+
+                return displayedSpriteText;
+            }
```

Comment:
a little verbose; can't this just be something of this sort?

```csharp
protected override OsuSpriteText CreateSpriteText() => displayedSpriteText = base.CreateSpriteText().With(text =>
{
    text.Spacing = new Vector2(-6);

    Winning = false;
});
```

Reply:
to work properly, it must be:
```csharp
protected override OsuSpriteText CreateSpriteText() => (displayedSpriteText = base.CreateSpriteText()).With(text =>
{
     text.Spacing = new Vector2(-6);
     Winning = false;
}
```
which to me, tbh doesn't sound much beneficial than the original one.

Reply:
this one maybe not too much as it assigns in the middle of it but the others I'd argue would benefit (didn't comment on those as not to hammer it home too much)